frontend/js/animation.js

```diff
@@ -41,8 +41,9 @@ export default function initAnimation(port) {
           maxDistance = 150,
           minDistance2 = minDistance * minDistance,
           maxDistance2 = maxDistance * maxDistance;
+
       var tau = 2 * Math.PI,
-          n = 20,
+          n = Math.max(Math.floor(width*height/15000), 20),
```

Comment:
Ok, will change it. The problem here is that the link radius doesn't scale at all. So at above 1920x1200 the particles don't have many links. But maybe this is a minor issue.